tests/deepspeed/test_deepspeed.py

```diff
@@ -153,6 +153,8 @@ def test_deepspeed_plugin(self, stage):
             zero_stage=3,
             offload_optimizer_device="cpu",
             offload_param_device="cpu",
+            offload_optimizer_nvme_path="/nvme",
```

Comment:
is this being tested?

Reply:
same for all the places of changes in the tests

Reply:
Should we add additional tests for this or keep it as it is?